config/environments/production.rb

```diff
@@ -9,57 +9,69 @@
   }
 
   config.cache_store = :dalli_store,
-  (ENV["MEMCACHIER_SERVERS"] || "").split(","),
-  {:username => ENV["MEMCACHIER_USERNAME"],
-   :password => ENV["MEMCACHIER_PASSWORD"],
-   :failover => true,
-   :socket_timeout => 1.5,
-   :socket_failure_delay => 0.2
-  }
+    (ENV["MEMCACHIER_SERVERS"] || "").split(","),
+    {
+      username: ENV["MEMCACHIER_USERNAME"],
+      password: ENV["MEMCACHIER_PASSWORD"],
+      failover: true,
+      socket_timeout: 1.5,
+      socket_failure_delay: 0.2
+    }
+
+    ActionMailer::Base.smtp_settings = {
+      port:           '587',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.